.github/scripts/oss-build.functions_tests.sh

```diff
@@ -134,4 +143,7 @@ assert_should_build_os "11.22.33" "22.22.33" "push" "All" "Error: OSS and EE ver
 assert_should_build_os "11.22.33" "11.22.33" "push" "All" "yes"
 assert_should_build_os "11.22.33" "11.22.00" "push" "All" "no"
 
+log_header "Tests for get_hz_dist_zip"
+assert_get_hz_dist_zip slim 5.4.0 -slim https://repo1.maven.org/maven2/com/hazelcast/hazelcast-distribution/5.4.0/hazelcast-distribution-5.4.0-slim.zip
```

Comment:
Maybe add more test cases

Reply:
0110272c997ee4deee4f2e93030069d707ef13b2

So other test cases (beyond negative ones - not sure how to do that) are jumping out right now.